kinder/pkg/kubeadm/kubednspatch.go

```diff
@@ -19,21 +19,13 @@ package kubeadm
 import (
```

Comment:
we forgot to remove kube-dns support from kinder.

here is an issue to track this:
https://github.com/kubernetes/kubeadm/issues/2484

Reply:
Yes, we can do this with another PR.